<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TwoPlayerMiniMaxWidget</class>
 <widget class="QWidget" name="TwoPlayerMiniMaxWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QPushButton" name="PlayButton">
     <property name="text">
      <string>Play</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="Depth">
     <item>
      <widget class="QLabel" name="MiniMaxDepthLabel">
       <property name="text">
        <string>Mini-MaxDepth: </string>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignRight">
      <widget class="QLineEdit" name="MiniMaxDepthInput">
       <property name="text">
        <string>6</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="PlayerColor">
     <item>
      <widget class="QLabel" name="PlayerColorLabel">
       <property name="text">
        <string>PlayerColor: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PlayerColor1Radio">
       <property name="text">
        <string>PlayerColor1</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="PlayerColor2Radio">
       <property name="text">
        <string>PlayerColor2</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
